src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ExportClient/BaseOtlpExportClient.cs

```diff
@@ -0,0 +1,53 @@
+// <copyright file="BaseOtlpExportClient.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using OpenTelemetry.Internal;
+
+namespace OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation.ExportClient
+{
+    /// <summary>Base class for sending OTLP export requests.</summary>
+    internal abstract class BaseOtlpExportClient
```

Comment:
Personally, I avoid using abstract classes for removing code duplication as much as possible.

I would rather create a `internal static EnsureUnencryptedSupportIsEnabled` in some helper static class and use it in `BaseOtlpGrpcExportClient` and `BaseHttpGrpcExportClient` constructors.

Why?

1. Code executed via inheritance is harder to follow/read/maintain
2. There is no need for polymorphism here (virtual methods etc) so why the use of inheritance?

_You can wait a day or two to check what other approvers/maintainers say. Maybe they disagree with my opinion. Therefore it is not a blocker and I am approving._

Reply:
I'm happy to use any of the approaches. How would you call the static class then? ExporterClientValidation.Ensure...?

Reply:
`ExporterClientValidation` works for me

Reply:
Done